src/preferences.html

```diff
@@ -46,68 +47,78 @@
                 <label class="switch"><input type="checkbox" id="repetition" name="repetition"><span class="slider round"></span></label>
             </div>
             <div class="flex-box mb-2">
-                <p data-i18n="$Preferences.minutesBetweenNotifications">Minutes between notifications</p>
+                <p><i class="fas fa-stopwatch"></i><span data-i18n="$Preferences.minutesBetweenNotifications">Minutes between notifications</span></p>
                 <input type="number" name="notifications-interval" id="notifications-interval" min="1" max="30" value="5" required onblur="this.value = this.checkValidity() ? this.value : 5">
             </div>
+            
+        </section>
+
+        <section>
+            <div class="section-title" data-i18n="$Preferences.appBehavior">App Behavior</div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.startOnLogin">Start on Login</p>
-                <label id='start-at-login' class="switch"><input type="checkbox" name="start-at-login"><span class="slider round"></span></label>
+                <p><i class="fas fa-globe"></i><span data-i18n="$Preferences.language">Language</span></p>
+                <select class="selects-right" id="language" name="language">
+                </select>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.countTodayOnTotals">Count Today in Totals</p>
-                <label id='count-today' class="switch"><input type="checkbox" name="count-today"><span class="slider round"></span></label>
-            </div>
-            <div class="flex-box mb-2">
-                <p data-i18n="$Preferences.overallBalanceStart">Overall Balance Start Date</p>
-                <label id='overall-balance-start-date'><input type="date" class="date-right" name="overall-balance-start-date"></label>
+                <p><i class="far fa-play-circle"></i><span data-i18n="$Preferences.startOnLogin">Start on Login</span></p>
+                <label id='start-at-login' class="switch"><input type="checkbox" name="start-at-login"><span class="slider round"></span></label>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.minimizeToTray">Minimize button should minimize to tray</p>
+                <p></i><span data-i18n="$Preferences.minimizeToTray">Minimize button should minimize to tray</span></p>
                 <label id='minimize-to-tray' class="switch"><input type="checkbox" name="minimize-to-tray"><span class="slider round"></span></label>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.closeMinimizeToTray">Close button should minimize to tray</p>
+                <p><span  data-i18n="$Preferences.closeMinimizeToTray">Close button should minimize to tray</span></p>
```

Comment:
How about:

```suggestion
                <p><i class="far fa-window-close"><span  data-i18n="$Preferences.closeMinimizeToTray">Close button should minimize to tray</span></p>
```
![image](https://user-images.githubusercontent.com/846063/97368129-0d709400-1889-11eb-8e9b-473e16c53b88.png)